src/components/CredentialCard/CredentialCard.tsx

```diff
@@ -137,6 +137,11 @@ function DeleteModal({
 }) {
   const t = browser.i18n.getMessage;
 
+  const explanation =
```

Comment:
I’d prefer to have this inline where it’s used… So that when I read this part of JSX I don’t have to jump to the definition.